dequelabs/cauldron
# test: switch tag tests to react testing library

Issue https://github.com/dequelabs/cauldron/issues/1159

This PR switches the `Tag` tests to React Testing Library. It also corrects the paths for `setupFilesAfterEnv` and `collectCoverageFrom` in the `jest.config.ts`.

## Validation steps
- Run `yarn test`
- See that there are coverage percentages in the table 
- Verify that the tests are converted properly
